content/guides/anomalies.md

```diff
@@ -101,6 +101,10 @@ Smaller window sizes will lead to faster alerts, however, with very small window
 
 Note that setting the window size to X minutes doesn't require an anomaly to last X minutes before an alert is triggered. You can tune the threshold to control how long an anomaly must last to trigger an alert. For example, with the window size set to 30 minutes, you can can alerted when an anomaly lasts for just five minutes by setting the threshold to 5/30 = 17%. That said, we have found that anomaly alerts are most reliable when the window size is between 15 minutes and an hour and the threshold is on the higher side (> 40%).
 
+### Why does `anomalies` not do anything in the dashboard? Why am I getting "No Data" for an Anomaly Alert?
```

Comment:
Instead of "not do anything", I'd try to be more explicit: "Why is `anomalies` not adding the gray predication band to my dashboard chart?"